packages/api/src/pins.js

```diff
@@ -144,7 +144,8 @@ async function createPin (normalizedCid, pinData, authTokenId, env, ctx) {
 
   await env.cluster.pin(cid, {
     name: pinName,
-    origins
+    origins,
+    metadata: pinMeta
```

Comment:
We can't send metadata to cluster it is not multi-tenant - what happens if someone else pins the same CID?

Reply:
It looks like its passed in NFT `pins-add`. Is that different?

Reply:
Nope, it should not be sent there either. Can you send a PR to fix? 🙏

Reply:
Ok cool. Removed it here, created an issue on NFT

[https://github.com/nftstorage/nft.storage/issues/1207](https://github.com/nftstorage/nft.storage/issues/1207)